src/consensus/dkg.rs

```diff
@@ -15,12 +15,16 @@ use crate::{
 };
 use bls::{PublicKeySet, SecretKeyShare};
 use bls_dkg::key_gen::{message::Message as DkgMessage, KeyGen};
+use lru_time_cache::LruCache;
 use std::{
     collections::{BTreeMap, BTreeSet, HashMap, VecDeque},
     fmt::{self, Debug, Formatter},
-    mem,
+    iter, mem,
+    time::Duration,
 };
 
+const OLDELDERS_EXPIRY_DURATION: Duration = Duration::from_secs(2 * 60);
```

Comment:
I would put underscore between `OLD` and `ELDERS`: `OLD_ELDERS_EXPIRY_DURATION`. It's more in line with rust style.